index.bs

```diff
@@ -1480,7 +1477,7 @@ are applicable only to regular attributes:
 
 <pre class="grammar" id="prod-AttributeRest">
     AttributeRest :
-        "attribute" Type AttributeName ";"
+        "attribute" TypeWithAttributes AttributeName ";"
```

Comment:
I agree `TypeWithExtendedAttributes` is long, but I think it would help make things clearer, here.